docs/develop/evm/introduction.md

```diff
@@ -0,0 +1,19 @@
+---
+id: introduction
+title: Project Aurora Introduction
+sidebar_label: Introduction
+---
+
+[Project Aurora](https://aurora.dev/) delivers an Ethereum Virtual Machine
+(EVM) on the NEAR protocol. It is comprised of the following components:
+
+| Name                                                                                    | Description                            |
+| ----------------------------------------------------------------------------------------| -------------------------------------- |
+| **[Aurora Engine](https://github.com/aurora-is-near/aurora-engine)**                    | Implements an EVM on the NEAR protocol |
+| **[Aurora CLI](https://github.com/aurora-is-near/aurora-cli)**                          | Command-Line Interface (CLI) for the Aurora Engine |
+| **[Aurora Engine Client for JS](https://github.com/aurora-is-near/aurora.js)**          | JavaScript Client for the Aurora Engine |
+| **[NEAR Protocol Web3 provider](https://github.com/aurora-is-near/near-web3-provider)** | Web3 provider to connect Ethereum frontend or Truffle to NEAR Protocol
+| **[Aurora Relayer](https://github.com/aurora-is-near/aurora-relayer)** | A JSON-RPC server compatible with Ethereum's Web3 API for NEAR's EVM environment
```

Comment:
Let's move the Relayer to precede the Web3 provider

Reply:
Done. I also changed the order of the corresponding pages accordingly.